src/workerd/server/tests/python/durable-object/worker.py

```diff
@@ -52,3 +69,11 @@ async def test(ctrl, env, ctx):
     if pyodide.__version__ != "0.26.0a2":
         res = await obj.jspi_method(9)
         assert res == 10
+
+    # Wait for alarm to get triggered.
+    while True:
```

Comment:
Sleep 200 ms and then make the request only once?

Reply:
I think that has a higher potential for being flaky. When running lots of tests at once the CI could easily make this alarm miss by multiple seconds.

Waiting indefinitely should prevent flaky false negatives at the expense of waiting a long time for the test to time out if the test is broken. That's preferable though.

Reply:
What about adding a 200ms sleep between the attempts?